mithril-common/src/crypto_helper/genesis.rs

```diff
@@ -63,13 +67,25 @@ impl ProtocolGenesisSigner {
         self.secret_key.sign(message).into()
     }
 
-    /// Export the secret key from the genesis verifier to a file. TEST ONLY
-    #[doc(hidden)]
-    pub fn export_to_file(&self, secret_key_path: &Path) -> StdResult<()> {
-        let mut genesis_secret_key_file = File::create(secret_key_path)?;
-        genesis_secret_key_file.write_all(self.secret_key.to_json_hex().unwrap().as_bytes())?;
+    /// Export the keypair from the genesis verifier to files
+    pub fn export_keypair_to_files(&self, keypair_path: &Path) -> StdResult<(PathBuf, PathBuf)> {
```

Comment:
Is it his responsibility to generate both at the same time or could we let that responsibility to the `create_and_save_genesis_keypair` of the `genesis.rs` in `tools` ?